pkg/cmd/roachprod/hosts.go

```diff
@@ -116,6 +116,9 @@ func loadClusters() error {
 		return err
 	}
 
+	debugDir := os.ExpandEnv(config.DefaultDebugDir)
```

Comment:
nit: put this in `initHostDir` and change the name of that function.

Reply:
Good call, done.